content/en/user-guide/ci/github-actions/samples/index.md

```diff
@@ -0,0 +1,396 @@
+---
+title: "Sample"
+linkTitle: "Sample"
+weight: 5
+description: >
+  Localstack Sample Project in Github Actions
+aliases:
+  - /ci/github-actions/samples
+tags: ["continuous-integration", "ci", "continuous-delivery", "testing","github-actions","sample"]
+---
+
+This page containes our Quickstart project's test pipeline implemented in Github actions.
+
+# Serverless image resizer s3 lambda
+This is an app to resize images uploaded to S3 in a serverless way.
+Find code in repo [here](https://github.com/localstack-samples/sample-serverless-image-resizer-s3-lambda).
+
+## Integration testing
+A build matrix for Ubuntu and MacOS to run integration tests triggered by various ways.
+In case of failure the pipeline dumps the logs of Localstack and preserves them as artifacts for further investigations.
```

Comment:
That's actually not true, the worfklow currently has a `if: always()` for the `Show localstack logs`. 
If you were refering to the diagnostic file, we should rephrase this part :) Also in that case it would be worth mentioning that the flag `LS_LOG=trace` needs to be set for the `setup-localstack` action.